pipelines/maven-zip-build/maven-zip-build.yaml

```diff
@@ -167,13 +167,15 @@ spec:
       workspace: workspace
   - name: sast-coverity-check
     params:
-    - name: image-url
+    - name: image-digest
       value: $(tasks.build-oci-artifact.results.IMAGE_URL)
+    - name: image-url
+      value: $(tasks.build-image-index.results.IMAGE_URL)
```

Comment:
Also we are mixing results from `tasks.build-oci-artifat` and `tasks.build-image-index`.  Is this correct?